pkg/upgrade/upgrade.go

```diff
@@ -222,8 +270,6 @@ func upgradeToVer3(cli *clientv3.Client, uctx Context) error {
 			// note that we lost CreateRevision, Lease, ModRevision, Version
 			ops = append(ops, clientv3.OpPut(newKey, string(kv.Value)))
 		}
-		// delete old key to provide idempotence
-		ops = append(ops, clientv3.OpDelete(pair.old.Path(), clientv3.WithPrefix()))
```

Comment:
Why do we delete these?

Reply:
for rollback to old version

Reply:
Seems may cause wrong bound relation for later upgrade

Reply:
revert